### PR TITLE
JAMES-2579 3.2.0 release

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by chibenwa** [comment]
capture d ecran de 2018-10-31 10-51-06

**Comment by chibenwa** [comment]
Call for vote:

```
Subject: Call for vote: Apache James 3.2.0

Hi,

I would like to propose the 3.2.0 release of the Apache James server.

Furthermore, you can see changes proposed to the website at the occasion
of that release, as well as communication on this GitHub pull request:
https://github.com/linagora/james-project/pull/1869

You can find:

 - The maven release staged in http://repository.apache.org/ as the artifact
#10XX 
 - 3.2.0 sources and binaries are uploaded on Apache dist. Please note
that we included JPA Guice and JPA SMTP guice.
 - The changelog for 3.2.0: https://github.com/chibenwa/james-project/blob/3.2.0-release-doc/CHANGELOG.md
 - The compatibility instructions/upgrade recommendation: https://github.com/chibenwa/james-project/blob/3.2.0-release-doc/upgrade-instructions.md

Voting rules:
 - This is a majority approval: this release may not be vetoed.
 - A quorum of 3 binding votes is required
 - The vote starts at XXXXX XXth of November 2018, 2am UTC
 - The vote ends at XXXXX XXth of November 2018, 2am UTC

You can answer to it just with +1 and -1. Down-votes may be motivated.

Cheers,

Benoit Tellier

```

File: src/homepage/_posts/2018-11-01-james-3.2.0.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- WebAdmin exposed mail re-indexing tasks
+- WebAdmin exposed health checks. This includes:
+   - Possibility to perform a single healthcheck, thanks to [mschnitzler](https://github.com/mschnitzler)
+   - Cassandra health checks thanks to [matzepan](https://github.com/matzepan)
+- IMAP MOVE commend (RFC-6851) On top of JPA. Thanks to [mschnitzler](https://github.com/mschnitzler)
+- JPA support for Sieve script storage thanks to [Sebast26](https://github.com/sebast26)
+- Sieve script upload via the CLI thanks to [matzepan](https://github.com/matzepan)
+- Mailet DOC: Exclude from documentation annotation thanks to [mschnitzler](https://github.com/mschnitzler)
+- **cassandra.pooling.max.queue.size** configuration option Thanks to [matzepan](https://github.com/matzepan)
+- **RecipentDomainI** and **SenderDomainIs** matchers by [athulyaraj](https://github.com/athulyaraj)
```
Comment (rouazana) [inline src/homepage/_posts/2018-11-01-james-3.2.0.markdown]: missing a s after RecipentDomainI

File: upgrade-instructions.md [hunk 2]
```diff
@@ -12,13 +12,39 @@ software documentation. Do not follow this guide blindly!
...
+As part of the SMTP protocol, a mail can be sent without sender. This was represented implicitly in James by a potentially null MailAddress
+(`null` or `MailAddress.nullSender()`). This means that mailet/matcher implementers needs to be aware, and handle these cases. This implicit
+handling makes nullSender hard to work with, and prooved to be error prone as part of the 3.2.0 development process.
+
+Hence we propose an alternative API returning a `MaybeSender` object, requiring the caller to explicitly handle missing sender.
+
+`Mail::getSender` had then been deprecated. We strongly encourage our users to rely on `Mail::getMaybeSender`.
+
+Note: thanks to java-8 default API methods, this is not a breaking change.
+
```
Comment (rouazana) [inline upgrade-instructions.md]: we should speak here about the deprecation also, no?

File: src/homepage/_posts/2018-11-01-james-3.2.0.markdown (hunk 1 above)
Comment (rouazana) [reply src/homepage/_posts/2018-11-01-james-3.2.0.markdown]: done

File: upgrade-instructions.md (hunk 2 above)
Comment (rouazana) [reply upgrade-instructions.md]: done

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**